### PR TITLE
add tslint rule to avoid the template strings if not interplating

### PR DESCRIPTION
**Pull request by prydonius**
This prevents strings like `hello` that don't actually do any
interpolation.

### CODE REVIEW

File: dashboard/tslint.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "rules": {
     "no-console": {
       "severity": "warning"
-    }
+    },
+    "quotemark": [true, "avoid-template"]
```
[the remark this answers is not in the document]
Comment (prydonius) [reply dashboard/tslint.json]: @andresmgot tslint didn't previously allow single quotes, is there any reason you want to allow single quotes?

**Comment by prydonius** [comment]
@andresmgot ah I guess you were referring to the issue where we were using single quotes for strings that contain double quotes, I just enabled the `avoid-escape` rule to allow this case.

**Comment by migmartri** [comment]
Are we enforcing the usage of either single or double quotes? eslint:recommended enforces it and one of the most common distributions prefers single vs double quotes https://github.com/airbnb/javascript#strings--quotes

**Comment by prydonius** [comment]
@migmartri this enforces usage of double quotes, I believe, but let me double check as it may be prettier that's convering those.

**Comment by prydonius** [comment]
It turns out in master we weren't enforcing double vs single, but prettier would automatically convert them to double. With the change in this PR, double quotes are now also enforced by tslint.

**Pull request merged**